### PR TITLE
Hih 32

### PR DESCRIPTION
**Pull request by nreddix**

### CODE REVIEW

File: model/schema.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "email" VARCHAR(255) NOT NULL,
+
+  PRIMARY KEY ( `id` ) 
+);
+
+CREATE TABLE "Connections" (
+  "friendID" INT( 11 ) AUTO_INCREMENT NOT NULL,
+  "friendNumber" INT ( 255) NOT NULL,
+  "friendEmail" INT ( 255) NOT NULL,
+  PRIMARY KEY ("id")
```
Comment (shaheinm) [inline model/schema.sql]: This table doesn't need a primary key or the AUTO_INCREMENT for friendID.  It's just a lookup table.  You should model it with Sequelize, and set the association in the User model.

Comment (shaheinm) [reply model/schema.sql]: Your model looks correct anyway, though

File: server.js [hunk 2]
```diff
@@ -24,14 +24,16 @@ app.use(express.static("public"));
...
-//   });
-// });
-app.listen(PORT, function() {
-      console.log("App listening on PORT " + PORT);
+db.sequelize.sync({ force: true }).then(function () {
+    app.listen(PORT, function () {
+        console.log("App listening on PORT " + PORT);
+        //   });
+        // });
+        app.listen(PORT, function () {
```
Comment (Corbineau) [inline server.js]: We only need one call to app.listen; go ahead and delete one of them.

File: model/user.js [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
-        type: Sequelize.STRING,
-        allowNull: false
-    },
-    email: {
-        type: Sequelize.STRING,
-        allowNull: false
-    },
+    });
+
+    // Post.associate = function (models) {
```
Comment (Corbineau) [inline model/user.js]:
```suggestion
    // User.associate = function (models) {
```

File: model/user.js [hunk 4]
```diff
@@ -1,23 +1,34 @@
...
+    //     // We're saying that a Post should belong to an Author
```
Comment (Corbineau) [inline model/user.js]:
```suggestion
    //     // This will actually be User to Relationships``

File: model/user.js [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
+    //     // A Post can't be created without an Author due to the foreign key constraint
```
Comment (Corbineau) [inline model/user.js:24]:
```suggestion

```

File: model/user.js [hunk 6]
```diff
@@ -1,23 +1,34 @@
...
+    //     Post.belongsTo(models.Author, {
```
Comment (Corbineau) [inline model/user.js]:
```suggestion
    //     User.hasMany(models.Relationship, {
```

File: model/relationship.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        relatedUserId: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+    });
+
+    // Post.associate = function (models) {
```
Comment (Corbineau) [inline model/relationship.js]:
```suggestion
    // Relationship.associate = function (models) {
```

File: model/relationship.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    //     // We're saying that a Post should belong to an Author
```
Comment (Corbineau) [inline model/relationship.js]:
```suggestion
    //     // We're saying that a Relationship belongs to a User
```

File: model/relationship.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    //     // A Post can't be created without an Author due to the foreign key constraint
```
Comment (Corbineau) [inline model/relationship.js]:
```suggestion
    // 
```

File: model/relationship.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    //     Post.belongsTo(models.Author, {
```
Comment (Corbineau) [inline model/relationship.js]:
```suggestion
    //     Relationship.belongsTo(models.User, {
```

Comment (Corbineau) [reply model/relationship.js]: I am not sure how this ought to change considering that it's not exactly a foreign key; it's two users from the same table relating to each other. ><

File: config/config.json [hunk 11]
```diff
@@ -0,0 +1,27 @@
+{
+    "development": {
+      "username": "root",
+      "password": null,
+      "database": "blogger",
```
Comment (shaheinm) [inline config/config.json]: Probably want to change `blogger` to your database name

**Review by Corbineau — CHANGES_REQUESTED** [review]
Suggestions included in the file; the main one is fixing the redundant call to the listener in the server file. The others are just useful, I think. 

note to everyone to run npm install after this is merged; there are new dependencies in package.json

**Review by InnanaDM — APPROVED** [review]
looks good, getting there.

**Review by InnanaDM — COMMENTED** [review]
almost ready to merge, I accidentally clicked approve

**Pull request merged**